.changeset/dry-trainers-own.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-extension-dsl-data-space-studio': patch
+'@finos/legend-extension-dsl-data-space': patch
+---
+
+Added form (FormTextEditor) and state management (FormEditorState) to DataSpace and plugin.
```

Comment:
only one set of changeset should be needed explaining the change

Reply:
deleted